### PR TITLE
Distribute scenario to all amoc nodes

### PR DESCRIPTION
**Pull request by telezynski**

### CODE REVIEW

File: src/amoc_api_scenarios_handler.erl [hunk 1]
```diff
@@ -128,21 +130,11 @@ to_json(Req0, State) ->
...
-                       ModuleName,
-                       ScenarioPath),
-            ResultBody = case Result of
-                             ok -> Result;
-                             {error, Errors, _Warnings} ->
-                                 R = io_lib:format("~p", [Errors]),
-                                 erlang:list_to_bitstring(lists:flatten(R))
-                         end,
+            %%Nodes = [amoc_slave:node_name(Node) || Node <- amoc_config:get(hosts, [])],
+            Nodes = [node()],
```
Comment (michalwski) [inline src/amoc_api_scenarios_handler.erl]: The list of nodes consists only of local nodes. Do we want to extend it by other nodes amoc nodes? In the `amoc_dist` there is not exproted function `amoc_nodes`. Maybe we can export it and use it here?

File: src/amoc_api_scenarios_handler.erl [hunk 2]
```diff
@@ -128,21 +130,11 @@ to_json(Req0, State) ->
...
+            io:format("**** Nodes: ~p~n", [Nodes]),
```
Comment (michalwski) [inline src/amoc_api_scenarios_handler.erl]: Do we want to keep this debug information, if so it'd be better to change it to ?LOG_NOTICE or ?LOG_INFO

File: src/amoc_api_scenarios_handler.erl (hunk 1 above)
Comment (telezynski) [reply src/amoc_api_scenarios_handler.erl]: I was hesitating what to do.
I think the best would be to export `amoc_nodes` and have one place where we will control how we want to calculate it.

File: src/amoc_api_scenarios_handler.erl (hunk 2 above)
Comment (telezynski) [reply src/amoc_api_scenarios_handler.erl]: To be deleted

**Comment by telezynski** [comment]
Added integration_test directory with scripts to build and run docker containers,
and also to run a test of installation of a scenario.
Test uses curl via HTTP API

**Pull request merged**